### PR TITLE
Clean Strings for url partials

### PR DESCRIPTION
**Pull request by robcolburn**
I had some difficulty with `test-cover`

```
Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}
```

Otherwise, let me know what you think :)

### CODE REVIEW

**Comment by helior** [comment]
I like! What do you think of renaming to `cleanPath`?

**Comment by robcolburn** [comment]
:+1: Thanks, couldn't decide on a name :)

**Comment by robcolburn** [comment]
Renamed

File: test/url.js [hunk 1]
```diff
@@ -63,3 +63,17 @@ describe('url.cleanString', function() {
     assert.equal(url.cleanString('    basketball', ['is', 'my', 'favorite'], {'name':'value'}, '   sport  '), 'basketball-sport');
   });
 });
+
+describe('url.cleanString', function() {
```
Comment (helior) [inline test/url.js]: `cleanPath`

**Comment by robcolburn** [comment]
Updated

**Comment by helior** [comment]
:+1:

**Pull request merged**